api/src/main/resources/config.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- <!DOCTYPE module PUBLIC "-//OpenMRS//DTD OpenMRS Config 1.0//EN" "http://resources.openmrs.org/doctype/config-1.0.dtd"> -->
+<module configVersion="1.0">
+
+	<!-- Base Module Properties -->
+	<id>@MODULE_ID@</id>
+	<name>@MODULE_NAME@</name>
```

Comment:
config.xml file was needed to be added as I used BaseContextSensitiveTest instead of BaseModuleContextSensitiveTest. This one also has been solved by now